net-print/cnijfilter-drivers/metadata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>mfld.fr@gmail.com</email>
+    <name>mfld-fr</name>
+  </maintainer>
+  <longdescription>
```

Comment:
Add the proxy-maint team as second maintainer so there will be someone to commit for you.

Reply:
Fixed